team-data.html

```diff
@@ -1,114 +1,81 @@
-        </div>
-        <div id = "mySidenav" class="sidenav">
-            <a href="javascript:void(0)" class="closebtn"
-            onclick = "closeNav()">&times;</a>
-            <a href="index.html">Home</a>
-            <a href="season-config.html">Season Configuration</a>
-            <a href="event-data.html">Event Data</a>
-            <a href="team-data.html">Team Data</a>
-            <a href="importing-exporting-data.html">Importing and Exporting Data</a>
-        </div>
+---
+layout: basic
+---
 
-        <div id= 'teamDataAlertModal' class='modal'>
-            <div class='modal-content' id = 'tDataModalContent'>
-                <span class='close' onclick = 'closeTDataModal()'>&times;</span>
-            </div>
-        </div>
+<div id= 'teamDataAlertModal' class='modal'>
+    <div class='modal-content' id = 'tDataModalContent'>
+        <span class='close' onclick = 'closeTDataModal()'>&times;</span>
+    </div>
+</div>
 
-        <h2 style="text-align:center">Team Data</h2>
-        <h3 style='text-align:center' id='tDataIntroText'></h3><br>
+<h2 style="text-align:center">Team Data</h2>
+<h3 style='text-align:center' id='tDataIntroText'></h3><br>
 
-        <h3 style='text-align:left' id='fieldsetTitle'></h3>
-        <fieldset>
-            <form id="teamRadioList" style="font-size:18px" onchange="changeCurrTeam()"></form>
-        </fieldset>
+<h3 style='text-align:left' id='fieldsetTitle'></h3>
+<fieldset>
+    <form id="teamRadioList" style="font-size:18px" onchange="changeCurrTeam()"></form>
+</fieldset>
 
-        <h3 style='text-align:center;text-decoration:underline' id='teamStatsTitle'></h3>
-    </body>
+<h3 style='text-align:center;text-decoration:underline' id='teamStatsTitle'></h3>
+</body>
 
-    <div class='w3-container'>
-        <fieldset>
-            <div class="w3-row ">
-                <a href="javascript:void(0)" onclick= 'displayTabContent(event, "pitTabContent");'>
-                    <div class=" w3-half w3-center w3-bottombar-maroon w3-hover-khaki w3-padding tabs">Pit Scouting</div>
-                </a>
-                <a href="javascript:void(0)" id = 'matchTab' onclick='displayTabContent(event, "matchTabContent")'>
-                    <div class=" w3-half w3-center w3-bottombar-maroon w3-hover-khaki w3-padding tabs">Match Scouting</div>
-                </a>
-            </div>
-        
-            
-            
-            
-            <div id='pitTabContent' style='display:none' class='w3-container w3-animate-fading tabContent'>
-                <h3>Pit Scouting</h3>
-                <h4>Here, enter and view answers to pit scouting questions. If you need to edit the questions, go to the 
-                    <a href="season-config.html">Season Configuration</a> page.<br><br>
-                    The answers you save will override previous answers.
-                </h4>
-                <!--<h4 id='questionConfirmation'></h4>-->
+<div class='w3-container'>
```

Comment:
You're not closing this div, and it's causing some alignment issues with the content above versus below.

Looking at the mobile simulator without it, everything looks good - so I think just remove it.